src/main/java/org/cascadebot/cascadebot/music/CascadePlayer.java

```diff
@@ -73,8 +74,12 @@ public void addTrack(AudioTrack track) {
         }
     }
 
-    public void loop(LoopType loop) {
-        this.loop = loop;
+    public void setLoopMode(LoopMode loopMode) {
```

Comment:
I prefered this being called loop as all the other methods for controlling the player are named what they do. we could maybe add `loopAll` and `loopSong` methods or just name it back to loop.

Reply:
Thing is, this is setting a property, all the other methods are called what they are because they are actions. This is literally `setting the loop mode` and so IMO it should be named as such due to Java convention.

Reply:
In that case everything else should be named differently witch doesn't make much sense either

Reply:
What else should be renamed? I don't see any other methods in this class that go against what I described

Reply:
Oh right i removed those methods, but still in this case It makes more sense to use the music term as it is a player imo, and this is why i suggest having other methods that make more sense in a music player context like loopAll

Reply:
That will then require extra work on the command side, how about just `loopMode` which is a comprise between the two 😛

Reply:
That works

Reply:
Wish we could use kotlin properties xD